README.rst

```diff
@@ -74,3 +74,9 @@ This repository uses `pre-commit`_ to ensure that all committed code follows min
 
 .. _Neuroglancer: https://github.com/google/neuroglancer
 .. _pre-commit: https://pre-commit.com/
+
+
+Acknowledgments
+===============
+
+`cloud-volume <https://github.com/seung-lab/cloud-volume>_` for compressed morton code and shard/minishard mask implementation.
```

Comment:
It would be good to mention the licence of that code (BSD 3-clause) and to also include the copyright of that code in the header of files that include it

Reply:
I can certainly add the license. 

With regard to the inclusion of copyright notice, the code snippet that were adopted were from: https://github.com/seung-lab/cloud-volume/blob/aebf4e2/cloudvolume/datasource/precomputed/image/common.py#L193-L198 (I am using the `aebf4e2` is the commit hash) and the result was https://github.com/HumanBrainProject/neuroglancer-scripts/blob/53e8042/src/neuroglancer_scripts/sharded_base.py#L167-L174 . 

I would ask that we do one of the following (roughly in the order of my preference):

1/ leave the file as is, but add copyright notice attribute to cloudvolume, with modification attribute to neuroglancer, similar to how cloud volume does adopts compressed segmentation code https://github.com/seung-lab/cloud-volume/blob/aebf4e2dcc89013b95c2510e43b8e98a850dcaaa/cloudvolume/py_compressed_segmentation.py OR

2/ separate the adopted code to a separate file (`compressed_morton.py` ?), and add the copyright notice in the separate file. OR

3/ leave the file as-is, but include at the header is adopted from the separate project. Inline, the copyright notice will be included, which complys with BSD3

I will for now work towards 1/, let me know if you disagree.

Reply:
I am completely fine with option 1, you could even additionally add a comment with a permalink to the snippet that was copied, for full traceability.